frontend/rogan-art-explorer/style.css

```
@@ -0,0 +1,229 @@
+.rogan {
+    --bg: #101513;
+    --surface: rgba(255, 255, 255, 0.06);
+    --border: rgba(255, 255, 255, 0.14);
+    --text: #f7f4ed;
+    --muted: #c2c7bf;
+    --accent: #e7b85f;
+    --accent2: #7fb6a0;
+    background: var(--bg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)